catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -715,7 +715,7 @@ public Response updateDocument(@PathParam("id") String id, @Context HttpHeaders
                 if (multipartBody != null) {
                     List<Attachment> contentParts = multipartBody.getAllAttachments();
                     if (contentParts != null && contentParts.size() > 0) {
-                        createInfo = parseAttachment(contentParts.get(0));
+                        createInfo = parseAttachments(contentParts, transformerParam);
```

Comment:
The `updateDocument` RESTEndpoint was not correctly parsing all attachments of the incoming request.